analyzers/its/expected/Ember-MM/generic.EmberCore.Extractor-{B413DEAF-4FDC-416A-B3E0-968C1C8F3F83}-S3900.json

```diff
@@ -0,0 +1,43 @@
+{
+"issues":  [
+{
+"id":  "S3900",
+"message":  "Refactor this method to add validation of parameter '_params' before using it.",
+"location":  {
+"uri":  "sources\Ember-MM\Addons\generic.EmberCore.Extractor\Module.Extractor.vb",
```

Comment:
The current behavior is fine. A single fix will deal with all of them.
You might have situation like
```
If SpecialParameter Then
  Argument.ToString ' Noncompliant that will be considered as FP because of SpecialParameter
Else 
  Argument.ToString ' FN that is a real issue would be hidden here
End If
```